v2/analyzer-deployment.yaml

```diff
@@ -26,7 +26,7 @@ spec:
         - name: VCAP_SERVICES_TONE_ANALYZER_API_KEY
           value:
         - name : VCAP_SERVICES_TONE_ANALYZER_TOKEN_ADDRESS
-          value: "iam.bluemix.net/identity/token"
+          value: "https://iam.bluemix.net/identity/token"
         - name: VCAP_SERVICES_TONE_ANALYZER_SERVICE_API
           value:
         - name: USE_HTTPS
```

Comment:
remove this too?

Reply:
this address is used to get the token, I just added https://

Reply:
yes, thanks!